.changeset/shaggy-zebras-kick.md

```diff
@@ -0,0 +1,8 @@
+---
+"@marigold/docs": patch
+"@marigold/components": major
+"@marigold/theme-b2b": patch
```

Comment:
Isn't that a major too?

Reply:
the theme didn't change? or what do you mean?